modulefiles/modulefile.ProdGSI.acorn

```diff
@@ -0,0 +1,33 @@
+#%Module######################################################################
+## NOAA-EMC/GSI
+##_____________________________________________________
+proc ModulesHelp { } {
+puts stderr "Set environment variables for NOAA-EMC/GSI"
+puts stderr "This module initializes the environment "
+puts stderr "for the Intel Compiler Suite $version\n"
+}
+module-whatis " NOAA-EMC/GSI whatis description"
+
+set COMPILER intel
+
+# Load modules
+module load envvar/1.0
+module load PrgEnv-intel/8.0.0
+module load intel/19.1.3.304
+module load intel/19.1.3.304/cray-mpich/8.1.4
+module load bacio/2.4.1
+module load bufr/11.4.0
+module load crtm/2.3.0
+module load ip/3.3.3
+module load nemsio/2.5.2
+module load prod_util/2.0.1
```

Comment:
It is better to update to latest version for prod_util

Reply:
Thanks Xianwu!  I have updated the modulefile to use prod_util/2.0.4 instead of prod_util/2.0.1 at 
db1a441.